### PR TITLE
Add QA jobs for COS milestone 62

### PR DESCRIPTION
**Pull request by adityakali**
@wonderfly Please review
cc/ @krzyzacy

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -2292,6 +2292,23 @@
       "UNKNOWN"
     ]
   },
+  "ci-kubernetes-e2e-gce-gci-qa-m62": {
+    "args": [
+      "--check-leaked-resources",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-gce-gci-qa-m62.env",
+      "--extract=gci/gci-61",
```
Comment (wonderfly) [inline jobs/config.json]: 62?

File: jobs/config.json [hunk 2]
```diff
@@ -2357,6 +2374,22 @@
       "UNKNOWN"
     ]
   },
+  "ci-kubernetes-e2e-gce-gci-qa-serial-m62": {
+    "args": [
+      "--check-leaked-resources",
+      "--env-file=jobs/platform/gce.env",
+      "--extract=gci/gci-61",
```
Comment (wonderfly) [inline jobs/config.json]: 62?

File: jobs/config.json [hunk 3]
```diff
@@ -2423,6 +2456,23 @@
       "UNKNOWN"
     ]
   },
+  "ci-kubernetes-e2e-gce-gci-qa-slow-m62": {
+    "args": [
+      "--check-leaked-resources",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-gce-gci-qa-slow-m62.env",
+      "--extract=gci/gci-61",
```
Comment (wonderfly) [inline jobs/config.json]: 62?

File: jobs/config.json (hunk 1 above)
Comment (adityakali) [reply jobs/config.json]: My bad. fixed.

File: jobs/config.json (hunk 2 above)
Comment (adityakali) [reply jobs/config.json]: Fixed.

File: jobs/config.json (hunk 3 above)
Comment (adityakali) [reply jobs/config.json]: Fixed.

File: jobs/env/ci-kubernetes-e2e-gce-gci-qa-m62.env [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# Use GCI builtin k8s version.
+
+### job-env
+GINKGO_PARALLEL=y
```
Comment (krzyzacy) [inline jobs/env/ci-kubernetes-e2e-gce-gci-qa-m62.env]: Can you replace the env with `--ginkgo-parallel=25` in config.json instead? Then we don't need the env files here.

**Comment by wonderfly** [comment]
LGTM once @krzyzacy's comment is resolved.

File: jobs/env/ci-kubernetes-e2e-gce-gci-qa-m62.env (hunk 4 above)
Comment (adityakali) [reply jobs/env/ci-kubernetes-e2e-gce-gci-qa-m62.env]: Done.

File: jobs/config.json [hunk 5]
```diff
@@ -2357,6 +2375,23 @@
...
+    "args": [
+      "--check-leaked-resources",
+      "--env-file=jobs/platform/gce.env",
+      "--extract=gci/gci-62",
+      "--gcp-project=e2e-gce-gci-qa-serial-m62",
+      "--gcp-zone=us-central1-f",
+      "--provider=gce",
+      "--test_args=--ginkgo.focus=\\[Serial\\]|\\[Disruptive\\] --ginkgo.skip=\\[Flaky\\]|\\[Feature:.+\\] --minStartupPods=8",
+      "--timeout=300m",
+      "--ginkgo-parallel=25"
```
Comment (wonderfly) [inline jobs/config.json]: This is the *serial* suite though

**Comment by krzyzacy** [comment]
You'll need to sort the args to pass the unit-test, otherwise LGTM

File: jobs/config.json (hunk 5 above)
Comment (adityakali) [reply jobs/config.json]: Ahh. Sorry. Too saturated to get even copy-paste right :(
Removed.

**Comment by krzyzacy** [comment]
(I'll fix the unit-test)

File: jobs/config.json [hunk 6]
```diff
@@ -2292,6 +2292,24 @@
       "UNKNOWN"
     ]
   },
+  "ci-kubernetes-e2e-gce-gci-qa-m62": {
+    "args": [
+      "--check-leaked-resources",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-gce-gci-qa-m62.env",
```
Comment (krzyzacy) [inline jobs/config.json]: oh - meh - remove the --env-file flags :-)

Comment (adityakali) [reply jobs/config.json]: yeah, done. realized it soon after.

**Pull request merged**